packages/storefront-lwc/src/modules/commerce/cart/cart.html

```diff
@@ -0,0 +1,204 @@
+<!--
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+<template>
+    <div role="main" id="maincontent">
+
+
+        <div class="cart-error-messaging cart-error">
+        </div>
+
+        <div class="container">
+            <h1 class="page-title">Your Cart</h1>
+            <div class="row cart-header">
+                <div class="col-sm-4 hidden-xs-down">
+                    <a class="continue-shopping-link" href="/s/RefArch/home?lang=en_US" title="Continue Shopping">
```

Comment:
probably want to clean up the href here.

Reply:
Done